docs/the-guide/chapter-6-orchestrate-the-workflow-with-a-cicd-pipeline/index.md

```diff
@@ -51,7 +51,7 @@ gcloud projects add-iam-policy-binding <id of your gcp project> \
 	--role="roles/viewer"
 
 # Create the Google Service Account Key
-gcloud iam service-accounts keys create ~/Downloads/google-service-account-key.json \
+gcloud iam service-accounts keys create ~/.config/gcloud/dvc-google-service-account-key.json \
```

Comment:
Did you have to create this directory or what it already here? If not, you can add a sentence to create the directory with `mkdir -p ~/.config/gcloud`.

Reply:
The path is created when installing `gcloud`. However, to in case this changes or it is not the same, I would add the following as you suggested just after the code block:

```markdown
!!! info
	The path `~/.config/gcloud` should be created when installing `gcloud`. If does
	not exist, you can create it by running `mkdir -p ~/.config/gcloud`
```

Reply:
Perfect, thanks!